db/seeds.rb

```diff
@@ -5,3 +5,68 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+racoons = [
+    {
+        id: 1,
```

Comment:
Rails wont let you work with custom ID's as it is in charge of ID's

Reply:
Thanks for that catch will do!